WebDriverAgentLib/Categories/XCUIDevice+FBHelpers.m

```diff
@@ -40,6 +63,48 @@ - (BOOL)fb_goToHomescreenWithError:(NSError **)error
   return YES;
 }
 
+- (BOOL)fb_lockScreen:(NSError **)error
+{
+  if (locked) {
+    return TRUE;
+  }
+  SEL mSelector = NSSelectorFromString(@"pressLockButton");
```

Comment:
You can add this method into private headers. There is no need to use NSInvokation. See https://github.com/facebook/WebDriverAgent/commit/e3134dbc3c5093de8ec244d92e1a79282667f2e8 for the reference